source/develop/release-management/features/network/manageiq_ovn.html.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Integrating OVN to ManageIQ
+category: feature
+authors: Alona Kaplan
+wiki_category: Feature
+wiki_title: Integrating OVN to ManageIQ
+wiki_revision_count: 1
+wiki_last_updated: 2017-07-09
```

Comment:
are the wiki_ lines needed? I'd guess they are ancient useless relics.

Reply:
removed